extensions/amp-story-interactive/0.1/amp-story-interactive.css

```diff
@@ -110,3 +110,94 @@
     opacity: .4;
   }
 }
+
+/* Disclaimer CSS rules */
+.i-amphtml-story-interactive-disclaimer {
+  position: absolute !important;
+  color: black !important;
+  bottom: calc(100% - 1.75em) !important;
+  right: 0 !important;
+  z-index: 1 !important;
+}
+
+.i-amphtml-story-interactive-container:not(.i-amphtml-story-interactive-active) .i-amphtml-story-interactive-disclaimer {
+  display: none !important;  /* Disable the disclaimer if not on active page to prevent tabbing */
+}
+
+.i-amphtml-story-interactive-disclaimer-alert {
+  width: 1.75em !important;
+  height: 1.75em !important;
+  position: absolute !important;
+  right: 0 !important;
+  bottom: 0 !important;
+  transform-origin: bottom right !important;
+  pointer-events: all !important;
+  background-color: var(--interactive-accent-color) !important;
+  border: none !important;
+  border-radius: 50% !important;
+  box-shadow: 0 .2em .2em 0 #00000040 !important;
+  padding: .25em !important;
+  cursor: pointer !important;
+  font-size: inherit !important;
+}
+
+.i-amphtml-story-interactive-disclaimer[active]
+  .i-amphtml-story-interactive-disclaimer-alert {
+  display: none !important;
+}
+
+.i-amphtml-story-interactive-disclaimer-alert::after {
```

Comment:
This could be part of `i-amphtml-story-interactive-disclaimer-alert`.
It's kind a big block but maybe it's better to not have he pseudo element in the dom.

.i-amphtml-story-interactive-disclaimer-alert {
  width: 1.75em !important;
  height: 1.75em !important;
  position: absolute !important;
  right: 0 !important;
  bottom: 0 !important;
  padding: .25em !important;
  font-size: inherit !important;
  transform-origin: bottom right !important;
  background-color: white !important;
  background-image: url('svg code') !important;
  background-repeat: no-repeat !important;
  background-position: center !important;
  background-size: 0.25em !important;
  border: .25em solid var(--interactive-accent-color) !important;
  border-radius: 50% !important;
  box-shadow: 0 .2em .2em 0 #00000040 !important;
  box-sizing: border-box !important;
}